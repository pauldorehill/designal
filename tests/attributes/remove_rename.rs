use designal::Designal;

#[derive(Designal)]
struct HumanBean {
    #[designal(remove, rename = "Yum")]
    taste: String,
}

fn main() {}
