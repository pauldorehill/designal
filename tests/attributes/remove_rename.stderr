error: You have removed and renamed a field
 --> $DIR/remove_rename.rs:5:16
  |
5 |     #[designal(remove, rename = "Yum")]
  |                ^^^^^^
